datakyt/db/create_table.sql

```diff
@@ -0,0 +1,152 @@
+SET ROLE datakyt_admin;
+
+-- Table description
+CREATE TABLE project
+(
+	id	        SMALLSERIAL 	PRIMARY KEY,
+	name		char(65)	    NOT NULL
+
+);
+
+
+CREATE TABLE office
+(
+	id	        SMALLSERIAL 	PRIMARY KEY,
+	city		char(255)	    NOT NULL
+
+);
+
+
+CREATE TABLE employee
+(
+	id		SMALLSERIAL 	PRIMARY KEY,
+	name		char(65)	NOT NULL,
+	email		char(65) 	UNIQUE,
+	phone		varchar(65)	UNIQUE,
+	project_id	SMALLSERIAL
+	    CONSTRAINT project_id_fk REFERENCES project (id) ON DELETE CASCADE,
+	office_id	SMALLSERIAL
+	    CONSTRAINT office_id_fk REFERENCES office (id) ON DELETE CASCADE
+
+
+);
+
+CREATE TABLE equipment_type
+(
+	id	SMALLSERIAL 	PRIMARY KEY,
+	name			char(255)	UNIQUE
+
+);
+
+CREATE TABLE equipment
+(
+	id			        SMALLSERIAL 	PRIMARY KEY,
+	name			    char(255)	NOT NULL,
+	warranty		    integer		NOT NULL,
+	cost			    money    	NOT NULL CHECK (cost > 0::money),
+	status			    varchar(65)	NOT NULL
```

Comment:
why 65? we know exactly the maximum length of the possible status value

Reply:
Okey, reduce to 10, max length